$OpenBSD: patch-vapi_r_util_vapi,v 1.1 2019/11/01 15:54:57 jasper Exp $

r_util.vapi:87.3-87.31: warning: the modifier `static' is not applicable to constants
                public static const string OS;
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
r_util.vapi:89.3-89.33: warning: the modifier `static' is not applicable to constants
                public static const string ARCH;
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Index: vapi/r_util.vapi
--- vapi/r_util.vapi.orig
+++ vapi/r_util.vapi
@@ -84,9 +84,9 @@ namespace Radare {
 		/* not supported */
 #else
 		[CCode (cname="R_SYS_OS")]
-		public static const string OS;
+		public const string OS;
 		[CCode (cname="R_SYS_ARCH")]
-		public static const string ARCH;
+		public const string ARCH;
 #endif
 		public static int sleep (int secs);
 		public static int usleep (int usecs);
